application/src/test/java/org/opentripplanner/updater/trip/TripUpdateBuilder.java

```diff
@@ -37,10 +37,10 @@ public TripUpdateBuilder(
     this.midnight = ServiceDateUtils.asStartOfService(serviceDate, zoneId);
   }
 
-  public TripUpdateBuilder addStopTime(String stopId, int minutes) {
+  public TripUpdateBuilder addStopTime(String stopId, int secondsFromMidnight) {
```

Comment:
It doesn't look like you're using the TripUpdateBuilder in this PR at all. What are these changes for?

(BTW, I agree that using minutes as the arguments was not very wise of me back then, however I prefer a time string like "08:12" these days.)

Reply:
It was used in a previous version of this PR but it is no longer required now. However the same change will appear again once I get the added / replacement PR in.